### ISSUE TITLE
[MU4 Issue]  "More"popup navigation is lost after escape and re-invoke 

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#9594]
**To Reproduce**
1. Navigate to More
2. Hit Enter - More popup opens
3. You can navigate inside More popup
4. Hit Escape to close popup 
5. Hit Enter again to navigate inside More popup  - **it doesn't work**

https://user-images.githubusercontent.com/90187801/139150471-daadcb1f-a1cd-4f2e-a1bd-7bd94e0fa56c.mp4

**Desktop:**

**Comment by Tantacrul** [comment]
Hey @DmitryArefiev - this seems to be working for me now. Can you recheck and reopen if needed?

**Comment by DmitryArefiev** [comment]
> Hey @DmitryArefiev - this seems to be working for me now. Can you recheck and reopen if needed?

@Tantacrul Yes, the same on my side (master build on Win10, Mac12, LinuxMint 20.2)

**Comment by HemantAntony** [comment]
I can't reproduce this in the latest master

**Comment by Tantacrul** [comment]
Neither can I. Well. I'm closing again, unless Dmitri is still having the issue.

**Comment by DmitryArefiev** [comment]
> Neither can I. Well. I'm closing again, unless Dmitri is still having the issue.

@Tantacrul Sorry for my English! 🙂
 In https://github.com/musescore/MuseScore/issues/9594#issuecomment-1119622950 I meant, I have the same result as yours. The issue no longer occurs in master
